klippy/extras/neopixel.py

```diff
@@ -42,25 +47,28 @@ def build_config(self):
         rmt = self.mcu.seconds_to_clock(RESET_MIN_TIME)
         self.mcu.add_config_cmd("config_neopixel oid=%d pin=%s data_size=%d"
                                 " bit_max_ticks=%d reset_min_ticks=%d"
-                                % (self.oid, self.pin, self.chain_count * 3,
+                                % (self.oid, self.pin, len(self.color_data),
                                    bmt, rmt))
         cmd_queue = self.mcu.alloc_command_queue()
         self.neopixel_update_cmd = self.mcu.lookup_command(
             "neopixel_update oid=%c pos=%c data=%*s", cq=cmd_queue)
         self.neopixel_send_cmd = self.mcu.lookup_query_command(
             "neopixel_send oid=%c", "neopixel_result success=%c", cq=cmd_queue)
-    def update_color_data(self, red, green, blue, index=None):
+    def update_color_data(self, red, green, blue, white, index=None):
         red = int(red * 255. + .5)
         blue = int(blue * 255. + .5)
         green = int(green * 255. + .5)
```

Comment:
Missing white = int(white *255.+.5)

Reply:
This removes the python error, but I think it's crashing my controller, I'm gonna reboot just to be sure..

Reply:
Working now, but the pixel is not RGBW it's GRBW :)